modules/navier_stokes/test/tests/finite_volume/cns/straight_channel_porosity_step/rotated-2d-bkt-function-porosity.i

```diff
@@ -0,0 +1,369 @@
+p_initial=1.01e5
+T=273.15
+# u refers to the superficial velocity
+u_in=1
+user_limiter='upwind'
+friction_coeff=10
+
+[GlobalParams]
+  fp = fp
+  two_term_boundary_expansion = true
+  limiter = ${user_limiter}
+[]
+
+[Mesh]
+  [cartesian]
+    type = GeneratedMeshGenerator
+    dim = 2
+    xmin = 0
+    xmax = 1
+    nx = 3
+    ymin = 0
+    ymax = 18
+    ny = 90
+  []
+[]
+
+[Modules]
+  [FluidProperties]
+    [fp]
+      type = IdealGasFluidProperties
+    []
+  []
+[]
+
+[Problem]
+  kernel_coverage_check = false
```

Comment:
```suggestion
```
I dont get what s wrong with coverage check here I guess. 
each variable has a kernel right?

Reply:
This branch has been in existence for a very long time ... well before you fixed up this integrity check